package.json

```
@@ -14,7 +14,9 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "db:setup": "cd ./src/server/ && knex migrate:latest && knex seed:run",
     "storybook": "cd ./src/client/ && start-storybook -p 3007",
-    "test": "jest"
+
+    "test": "jest --watch"
```

```suggestion
    "test:watch": "jest --watch"
```
Please don't modify the `npm test` command as it will be run by Travis as well.